libs/design-system/buttons/text-hyperlink/src/lib/directives/text-hyperlink.directive.ts

```diff
@@ -18,3 +18,22 @@ import { ClickEventDirective } from '@hra-ui/common/analytics';
   },
 })
 export class TextHyperlinkDirective {}
+
+/**
+ * Applies hyperlink styles when placed on a <button> tag.
+ * Styles the button as a text hyperlink with underline.
+ * Also attaches a click event for analytics.
+ */
+@Directive({
+  selector: 'button[hraTextButton]',
+  hostDirectives: [
+    {
+      directive: ClickEventDirective,
+      inputs: ['hraClickEvent: hraTextButton'],
+    },
+  ],
+  host: {
+    class: 'hra-text-hyperlink-button',
+  },
+})
+export class TextHyperlinkButtonDirective {}
```

Comment:
Please rename to `TextButtonDirective` and move to it's own file (`text-button.directive.ts`)